src/pages/payment-methods/index.tsx

```diff
@@ -291,68 +255,106 @@ const DisplayAccordianItem = ({ pd, locale }: PaymentMethodsProps) => {
+                                            <Localize translate_text="_t_Deposit processing time_t_" />
+                                        </BoldText>
                                     </Th>
                                 ) : pd.is_dp2p ? (
                                     <Th>
-                                        <BoldText>{localize('Daily withdrawal limits')}</BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_Daily withdrawal limits_t_" />
+                                        </BoldText>
                                     </Th>
                                 ) : (
                                     <Th>
-                                        <BoldText>{localize('Deposit')}</BoldText>
-                                        <BoldText>{localize('processing time')}</BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_Deposit_t_" />
+                                        </BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_processing time_t_" />
+                                        </BoldText>
                                     </Th>
                                 )}
 
                                 {!pd.is_fiat_onramp && !pd.is_dp2p && (
                                     <Th>
-                                        <BoldText>{localize('Withdrawal')}</BoldText>
-                                        <BoldText>{localize('processing time')}</BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_Withdrawal_t_" />
+                                        </BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_processing time_t_" />
+                                        </BoldText>
                                     </Th>
                                 )}
                                 {pd.is_dp2p && (
                                     <Th>
-                                        <BoldText>{localize('Processing time')}</BoldText>
+                                        <BoldText>
+                                            <Localize translate_text="_t_Processing time_t_" />
+                                        </BoldText>
                                     </Th>
                                 )}
                                 <Th>
                                     <BoldText>
-                                        {pd.is_dp2p ? localize('More info') : localize('Reference')}
+                                        {pd.is_dp2p ? (
+                                            <Localize translate_text="_t_More info_t_" />
```

Comment:
you can create a constant for this,

`const sth = pd.is_dp2p ? '_t_More info_t_' : '_t_Reference_t_'
`

`<Localize translate_text={sth} />`